LTOssian/grader
# 34-Grade Form

## What was done
- [x] Shared postman collection as json for the user to import
- [x] Fixed some endpoints to not rely on group_id and be used in the `/generate` endpoints, such as grade creation and get grade form information
- [x] Create service method for GET Grade Form Informations and Post Grade
- [x] Create Component for Grade Form
- [x] Improved docuentation

## How to test
- Import the `Grader-Louisan-Tchitoula.postman_collection` file to your own postman and see the endpoints available.
